docker/statutory/Dockerfile

```diff
@@ -1,28 +1,31 @@
+# FIXME use alpine
 FROM node:14.15.4
 
 RUN mkdir -p /usr/app/src \
 	&& mkdir -p /usr/app/media \
 	&& mkdir -p /usr/app/scripts
 
-RUN apt-get update
-RUN apt-get install netcat -y
+WORKDIR /usr/app/src
 
-COPY ./docker/statutory/bootstrap.sh /usr/app/scripts/bootstrap.sh
-COPY ./docker/statutory/wait.sh /usr/app/scripts/wait.sh
-COPY . /usr/app/src
+COPY package.json /usr/app/src
 
 RUN chown -R node:node /usr/app
 
-WORKDIR /usr/app/src
-
 USER node
 
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 ENV PATH="/home/node/.npm-global/bin:${PATH}"
 
-RUN npm install -g --loglevel warn nodemon bunyan && npm cache clean --force
-RUN npm install --loglevel warn
+# FIXME remove nodemon for production
+RUN npm install -g nodemon@2.0.7
+RUN npm install -g bunyan@1.8.15
+RUN npm cache clean --force
+RUN npm install
+
+COPY --chown=node:node ./docker/statutory/bootstrap.sh /usr/app/scripts/bootstrap.sh
+COPY --chown=node:node . /usr/app/src
+RUN mkdir /usr/app/src/state
 
-CMD sh /usr/app/scripts/bootstrap.sh && nodemon -e "js,json" lib/run.js
+CMD ["/usr/app/scripts/bootstrap.sh"]
```

Comment:
we can put `nodemon` to the bootstrap script as well and update it so it'd be something like that:

```
echo "Creating database..."
npm run db:create
echo "Migrating database..."
npm run db:migrate
echo "Running app..."
nodemon -e "js,json" lib/run.js
```

but this won't allow overriding the command from docker-compose.yml's arguments

Reply:
Does the bootstrap run in prod as well? Because we want to remove nodemon from running in prod

Reply:
yes it is, we migrate the DB the same way in production as in development.
I still don't see the benefits of having a single command in CMD instead of having 2 commands, like it was before